applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -1366,9 +1367,14 @@ get_caller_id_number(Call) ->
     CallerIdNumber = whapps_call:caller_id_number(Call),
     case whapps_call:kvs_fetch('prepend_cid_number', Call) of
         'undefined' -> CallerIdNumber;
-        Prepend -> <<(wh_util:to_binary(Prepend))/binary, CallerIdNumber/binary>>
+        Prepend -> Pre = <<(wh_util:to_binary(Prepend))/binary, CallerIdNumber/binary>>,
+                   truncate_caller_id(Pre)
     end.
 
+-spec truncate_caller_id(ne_binary()) -> ne_binary().
+truncate_caller_id(Prepend) ->
+    <<Prepend:35/binary>>.
```

Comment:
I'm confused on how this is supposed to work:

```erlang
5> P = <<"abc">>.
<<"abc">>
6> <<P:35/binary>>.
** exception error: bad argument
```

So what happens when the prepended caller ID isn't 35 characters? Or am I missing a step earlier?